tracker-agent/src/main/resources/application.properties

```diff
@@ -1,6 +1,5 @@
 application.name=tracker-agent
-server.port=8689
-application.key=tracker-agent-application-key
+application.key=tracker-agent-key
```

Comment:
why server.port removed? It must be still needed for standalone installations to not run on different port (8080) right? Don't we support standalone installations anymore for tracker-agent?

Reply:
This knowledge is lost together with the author of this code. I'll add the port back.